lib/sidekiq/rails.rb

```diff
@@ -60,4 +66,8 @@ def inspect
       end
     end
   end
+
+  ActiveSupport::BacktraceCleaner.class_eval do
```

Comment:
Do you need to require this, just in case?

Reply:
Seems like it is not necessary as it is always required. In newly created apps `rails` is required - https://github.com/sidekiq/sidekiq/blob/1813cbdbd5e2fff23bfed5488ebcf5c6eeebb2e2/myapp/config/application.rb#L3 and from there backtrace cleaner is loaded https://github.com/rails/rails/blob/9180e61802d1c025f15bf57db92f392400dce46a/railties/lib/rails.rb#L16

Reply:
And even `rails` is already required from this file at the top.

Reply:
Instead of aliasing the method, maybe we could wrap setting the config into a proc? e.g.

```ruby
config[:backtrace_cleaner] = -> backtrace { ::Rails.backtrace_cleaner.clean(backtrace) }
```

Reply:
Yeah, that will work too 👍

Reply:
Cool, hopefully this should do it: https://github.com/sidekiq/sidekiq/pull/5796. Thanks again!